### ISSUE TITLE
Review all emails that are sent for CNCF and update design

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#34]
After we launch the new site, we should audit all regular emails sent out by CNCF, like the "CNCF webinars this week" email, and update the design to use the new design patterns from cncf.io.

- [ ] Audit and list all emails sent out by CNCF
- [ ] Pick which ones are most in need of a design lift and get approval from key stakeholders
- [ ] Design and implement better designs

**Comment by cjyabraham** [comment]
Consider [designing emails within WP using Gutenberg](https://vimeo.com/401044615/ea9de156a0).

**Comment by cjyabraham** [comment]
All emails are currently being migrated into HubSpot so we should see if we can insert ourselves into that migration so that we can improve their design at the same time.

**Comment by cjyabraham** [comment]
Waiting on access to HubSpot

**Comment by cjyabraham** [comment]
closing in favor of #445
